src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -41,6 +41,14 @@
                 <Setter Property="FontSize" Value="12"/>
                 <Setter Property="TextAlignment" Value="Left"/>
                 <Setter Property="PagePadding" Value="0"/>
+
+                <Style.Resources>
+                    <Style TargetType="Image">
+                        <Setter Property="Stretch" Value="UniformToFill" />
+                        <Setter Property="StretchDirection" Value="DownOnly" />
+                        <Setter Property="MaxWidth" Value="{Binding ActualWidth, ElementName=bodyMarkdown}" />
```

Comment:
I suspect here that using a `RelativeSource AncestorType` binding might be the best way to go: http://stackoverflow.com/questions/84278/how-do-i-use-wpf-bindings-with-relativesource